src/index.js

```diff
@@ -90,29 +90,25 @@ function buildArgs(options) {
 // function creates the conf object to be written to that temp file before
 // passing to jsdoc as a command line argument.
 function buildConf(options) {
-    let opts = options || {};
-
+    const opts = options || {};
+    const config = _.isPlainObject(opts) ? opts.config : {};
```

Comment:
Opted to use `_.isType` instead of `typeof` check to let lodash handle the nuances.